notebooks/en/Prompt_Tuning_PEFT.ipynb

```diff
@@ -0,0 +1,7324 @@
+{
```

Comment:
<p>"I want you to act as an English translator," -&gt; "I want you to act as a motivational coach. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>ups... modified! </p>